src/server/scripts/World/action_ip_logger.cpp

```diff
@@ -104,22 +104,22 @@ class AccountActionIpLogger : public AccountScript
             switch (aType)
             {
             case ACCOUNT_LOGIN:
-                systemNote = "Logged on Successful AccountLogin";
+                systemNote = "Logged into WoW";
```

Comment:
I think it would be practical to have a more general log entry, while still being specific enough.
```suggestion
                systemNote = "Game client account login successful.";
```
(if you have got a better suggestion, feel free to use it)